src/views/Pipeline.vue

```diff
@@ -495,6 +530,24 @@ export default defineComponent({
         this.isJobDetailAnimationCompleted = true;
       }
     },
+    async updatePinnedJobs(enumId: any) {
+      if(this.getPinnedJobs?.searchPrefId) {
+        const payload = {
+          "searchPrefId": this.getPinnedJobs?.searchPrefId,
```

Comment:
I think we should pass the pinned the job list to actions and action taking care of checking if there is already a search pref

Reply:
Updated code Sir.